pkg/package/manifest.go

```diff
@@ -210,6 +236,11 @@ func parseVariableSchema(manifest []byte) map[string]Schema {
 			if description, okk := vv["description"].(string); okk {
 				schema.Description = description
 			}
+
+			if dflt, okk := vv["default"]; okk {
+				b, _ := json.Marshal(dflt)
+				schema.Default = strings.TrimPrefix(strings.TrimSuffix(string(b), "\""), "\"")
```

Comment:
nit: I usally prefer to put each call on a separete line rather than nest functions, it makes it hard to decipher what is going on here,